src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -85,32 +88,10 @@ public Task GetRepliationStats()
             using (ContextPool.AllocateOperationContext(out context))
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
-                var incomingStats = new DynamicJsonArray();
-                var outgoingStats = new DynamicJsonArray();
-                var resovlerStats = new DynamicJsonArray();
-
-                foreach (var e in stats.IncomingStats)
-                {
-                    incomingStats.Add(e.ToJson());
-                }
-
-                foreach (var e in stats.OutgoingStats)
-                {
-                    outgoingStats.Add(e.ToJson());
-                }
-
-                foreach (var e in stats.ResolverStats)
-                {
-                    resovlerStats.Add(e.ToJson());
-                }
-
-                context.Write(writer, new DynamicJsonValue
-                {
-                    ["LiveStats"] = stats.LiveStats(),
-                    ["IncomingStats"] = incomingStats,
-                    ["OutgoingStats"] = outgoingStats,
-                    ["ResovlerStats"] = resovlerStats
-                });
+                stats.Update();
+                var entityToBlittable = new EntityToBlittable(null);
```

Comment:
This is fine here, since this isn't perf sensitive, but be aware that in hotspots, we can't pay for this.